package.json

```diff
@@ -43,6 +43,19 @@
     "stylelint": "^9.1.1",
     "stylelint-config-recommended": "^2.1.0"
   },
+  "jest": {
+    "collectCoverageFrom": [
```

Comment:
Could this be inverted so for now it _only_ calculates coverage in the directory we've tested?